src/components/WhatWeDo/Blurbs.js

```diff
@@ -58,7 +58,7 @@ class Blurbs extends Component {
             A percentage of your purchase is immediately contributed to helping people in need when using our app.
           </h3>
 
-          <img className = "RightBlurbImages" src={require("../../img/img-clipart-wwd-percent.png")} alt="percentage"/>
+          <img className = "RightBlurbImages" id = "wwdRightBlurb" src={require("../../img/img-clipart-wwd-percent.png")} alt="percentage"/>
```

Comment:
WhatWeDo/Blurbs.js: Code is clean. Good use of `id = "wwdLeftBlurb"` and  `id = "wwdRightBlurb"` to present the images.